src/frontend/src/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageViewContainer.tsx

```diff
@@ -155,6 +171,32 @@ const WorkPackageViewContainer: React.FC<WorkPackageViewContainerProps> = ({
     </div>
   );
 
+  const tabs = (
```

Comment:
can we use our tab component here?

Reply:
tab component?

Reply:
Yes we should havw a component for tabs

Reply:
Do you mean you want me to make one? If you're talking about an existing tab component it's not in the components folder

Reply:
we have one thats for the projects maybe abstract that?